app/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "headlamp",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "copy-server-binary": "cd ../ && make backend && cp -r ./backend/server ./app/electron/",
+    "start-electron": "npm run copy-server-binary && cd ../ && make run-backend  & cd ../frontend/ && export BROWSER=none && npm start",
+    "build-electron": "npm run copy-server-binary && cd ../frontend/ && npm run build && cp -r ../app/electron/. build/electron",
+    "package-electron": "mkdir -p build && cp -r ../frontend/build/ ./build && electron-builder build --mac --win --linux -c.extraMetadata.main=build/electron/main.js --publish never",
+    "serve-electron-dev": "export ELECTRON_START_URL=http://localhost:3000 && cd ../frontend/ && cp -r ../app/electron/. electron && electron ."
+  },
+  "build": {
+    "appId": "com.kinvolk.headlamp",
+    "extraMetadata": {
+      "main": "build/electron/main.js"
+    },
+    "files": [
+      "build/**/*",
+      "node_modules/**/*"
+    ],
+    "extraResources": "./electron/server",
+    "publish": {
+      "provider": "github",
+      "repo": "headlamp",
+      "owner": "kinvolk"
+    },
+    "extends": null
+  },
+  "keywords": [],
+  "author": "kinvolk",
+  "dependencies": {
+    "@material-ui/core": "^4.7.2",
+    "@material-ui/lab": "^4.0.0-alpha.38",
+    "@types/javascript-time-ago": "^2.0.1",
+    "@types/jest": "^25.2.1",
+    "@types/lodash": "^4.14.150",
+    "@types/node": "^13.13.2",
+    "@types/react": "^16.9.34",
+    "@types/react-dom": "^16.9.6",
+    "@types/react-redux": "^7.1.7",
+    "@types/react-router-dom": "^5.1.5",
+    "@types/webpack-env": "^1.15.2",
+    "ansi-to-react": "^5.1.0",
+    "javascript-time-ago": "^2.0.4",
+    "js-base64": "^2.5.1",
+    "js-cookie": "^2.2.1",
+    "js-yaml": "^3.14.0",
+    "lodash": "^4.17.15",
+    "monaco-editor": "^0.19.0",
+    "notistack": "^0.9.7",
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0",
+    "react-hotkeys-hook": "^2.2.2",
+    "react-monaco-editor": "^0.32.1",
+    "react-redux": "^7.1.3",
+    "react-router-dom": "^5.1.2",
+    "react-scripts": "^3.4.1",
+    "react-scroll": "^1.7.14",
+    "recharts": "^1.8.5",
+    "redux": "^4.0.4",
+    "redux-saga": "^1.1.3",
+    "swagger-parser": "^8.0.3",
+    "typescript": "^3.8.3",
+    "xterm": "^4.3.0"
```

Comment:
I think we don't need all this (which is problematic as now we need to keep this and the frontend's package.json in sync) if we just build the frontend in its own folder rather than copying the whole project here. i.e. in the `build-electron` command above, we'd just go into frontend, build it, and copy it. Something like `pushd ../frontend/; npm build; popd; mkdir -p build; cp -R ../frontend/build/* ./build/`